app/scripts/modules/google/instance/details/instanceDetails.html

```diff
@@ -172,7 +172,18 @@ <h3 select-on-dbl-click>
         </dd>
       </dl>
     </collapsible-section>
-    <collapsible-section heading="SSH">
+    <collapsible-section heading="Labels">
+      <div ng-if="!instance.labels.length">No labels associated with this instance</div>
+      <dl ng-if="instance.labels.length">
+        <dt ng-repeat-start="label in instance.labels">
```

Comment:
For here (and another template in this PR), could you do an `ng-repeat-start="(key, value) in instance.labels"`? It would save you from doing the translation from map to list in both controllers.